test/colony-network.js

```diff
@@ -256,20 +270,23 @@ contract("ColonyNetwork", accounts => {
 
   describe("when adding a skill", () => {
     beforeEach(async () => {
-      await colonyNetwork.createColony("Common Colony", ...TOKEN_ARGS);
+      const token = await Token.new(...TOKEN_ARGS);
+      await colonyNetwork.createColony(COLONY_KEY, token.address);
+      const colony = await colonyNetwork.getColony.call(COLONY_KEY);
+      await token.setOwner(colony);
```

Comment:
Why isn't this happening in the `createColony` call when the token is assigned to the colony?

Reply:
Because ColonyNetwork is not the owner of the token, so it cant assign another owner. It think it cant be done with `delegatecall` solidity function either, since i need to change the storage of the `DSAuth` contract, but i need to preserve the `msg.sender`. Do you have a suggestion what is the best way to do this?

Reply:
After a discussion with @area he made a good point that ERC20 is not `ownable` so let's leave it as you've implemented and rely on Token owners to set the Token ownership correctly.